src/pages/User/User.js

```diff
@@ -1,7 +1,108 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
+import styled from 'styled-components';
+import Filter from './Components/Filter';
+import Item from '../../components/ProjectCard/ProjectCard';
+import { debounce } from 'lodash';
 
 function User() {
-  return <></>;
+  const [likeList, setLikeList] = useState([]);
+  const [pushList, setPushList] = useState([]);
+  const [searchInput, setSearchInput] = useState({
+    searchedLike: '',
+    searchedPush: '',
+  });
+
+  const handleSearchInput = e => {
+    const { name, value } = e.target;
+    setSearchInput({ ...searchInput, [name]: value });
+  };
+
+  const { searchedLike, searchedPush } = searchInput;
+
+  const filteredLike = likeList?.filter(item =>
+    item.title.includes(searchedLike)
+  );
+  const filteredPush = pushList?.filter(item =>
+    item.title.includes(searchedPush)
+  );
```

Comment:
```suggestion
  const filteredLike = likeList.filter(item =>
    item.title.includes(searchedLike)
  );
  const filteredPush = pushList.filter(item =>
    item.title.includes(searchedPush)
  );
```
어차피 초기값 빈 배열이라 에러 안날거 같은데, 그러면 옵셔널 체이닝 불필요하지 않나요?

Reply:
오호..! 네 에러가 안납니다!